lib/Crate.js

```diff
@@ -113,6 +114,15 @@ Crate.prototype.plugin = function(schema, options) {
       })
     }
 
+    // build suggested target storage location
+    var format = options.fields[field].url || options.url || ':modelName/:fieldName/:id'
+    tasks.push(function(callback) {
+      targetBuilder(this, field, attachment, format, function (error, target) {
```

Comment:
I'm using `this` here instead of model, because `model` is something other than a model for array type fields.